packages/x-data-grid-premium/src/index.ts

```diff
@@ -1,13 +1,12 @@
 import './typeOverloads';
 
-export {
-  /**
-   * @deprecated
-   * Use `@mui/x-license` package instead:
-   * import { LicenseInfo } from '@mui/x-license';
-   */
-  LicenseInfo,
-} from '@mui/x-license';
+import { LicenseInfo as LicenseInfoExport } from '@mui/x-license';
+/**
+ * @deprecated Use `@mui/x-license` package instead:
+ * @example import { LicenseInfo } from '@mui/x-license';
+ */
+export const LicenseInfo = LicenseInfoExport;
```

Comment:
WDYT about doing the following to avoid the change from `class` to `variable`? 🤔 
```suggestion
export class LicenseInfo extends LicenseInfoExport {}
```